Events/LocationEvent.cs

```diff
@@ -19,6 +19,7 @@ static LocationEvent()
             VARIABLES.Add("x", "The X co-ordinate of the system in which the commander resides");
             VARIABLES.Add("y", "The Y co-ordinate of the system in which the commander resides");
             VARIABLES.Add("z", "The Z co-ordinate of the system in which the commander resides");
+            VARIABLES.Add("distancefromstar", "The distance from the main star (unless close)");
```

Comment:
What do you mean by `(unless close)`?
Can we simply use `distance`? (to match the `body` object).

Reply:
“Unless close” is direct from the journal manual.

Reply:
Interesting. 50,000 ls out from the main star doesn't seem particularly close to me, but it's still not being written at that distance.

Reply:
Changed description to `"The distance of the nearest body (when present) from the main star"`